Dockerfile

```diff
@@ -0,0 +1,11 @@
+from ubuntu:22.04
+
+run apt-get update 
+run apt-get install git python3 python3-pip -y
+run apt-get clean
+
+run pip3 install git+https://github.com/informatyzacja-sspwr-projekty/CertMinder.git
+
+copy certminder.yml /certminder.yml
+
+cmd certminder -s -c
```

Comment:
change `run` -> `RUN`, `from` -> `FROM`, ... to comply with common Docker standards

Reply:
Yes, please!